2.3/Dockerfile.rhel7

```diff
@@ -0,0 +1,41 @@
+FROM rhscl/s2i-base-rhel7
```

Comment:
it failed our tests as soon as i merged it (we run the same tests when building from master, prior to pushing to dockerhub), so the tests seem correct, but somehow it slipped through our PR tester...maybe related to the other issues we were having.